gemfiles/rails-6.0.gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gemspec path: "../"
+gem "railties", "6.0.0.beta3"
```

Comment:
Please change this to:

`">= 6.0.0.beta3", "< 6.1"`

Reply:
Changed. Thank you for reviewing.